.travis.yml

```diff
@@ -34,14 +34,13 @@ before_install:
     fi;
 
 install:
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
-      wget http://download.qt.io/official_releases/online_installers/qt-unified-linux-x64-online.run ;
-      chmod a+x ./qt-unified-linux-x64-online.run ;
-      QT_QPA_PLATFORM=minimal QTVER="${QTVER//.}" travis_wait 60 ./qt-unified-linux-x64-online.run --script qt-installer-noninteractive.qs --no-force-installations --verbose ;
-    fi;
-
+  # Install QT from a PPA
   # Install updated libglew-dev since the version provided by trusty is outdated
+  # TODO: Install QT for OS X when time comes
   - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+      sudo add-apt-repository ppa:beineri/opt-qt-5.10.1-trusty -y;
```

Comment:
why not use ${QTVER}

Reply:
Yeah Im moving that to the bottom where it should be I'll see if that works there